src/components/delete-link/deleteLink.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+angular.module("openshiftCommonUI")
+  .directive("deleteLink", function ($uibModal, $location, $filter, $q, hashSizeFilter, APIService, DataService, AlertMessageService, Logger) {
```

Comment:
There's a lot of code in this directive for handling other types and then special handling for projects. I wonder if we shouldn't have a separate, simpler directive just for projects and leave the other in origin-web-console.